tests/api/test_dataset_location.py

```diff
@@ -70,10 +73,8 @@ def test_get_dataset(client, dataset1, env1, user, group):
 
 
```

Comment:
What is the advantage of using MagicMock?

Reply:
Honestly? it's just working :) 
I couldn't find a better way to mock _instance_ of class. Tried a few other ways to patch an instance, but got only `FAILED`. This one worked)